submissions/kt-std/js-dom/js/script.js

```diff
@@ -0,0 +1,113 @@
+        period: '687 Earth days',
+        position: '04'
+    }
+];
+
+
+document.querySelector("#menu-btn").addEventListener('click', function display(e) {
+    document.querySelector("#menu").classList.toggle("visible");
+});
+
+document.querySelector("#menu").addEventListener('click', function display(e) {
+    let sectionName = e.target.innerHTML.toLowerCase(),
+        target = e.target;
+    index = CONTENT.findIndex(el => el.heading.toLowerCase() == sectionName);
+    setData(index);
+});
+
+let addAnimation = (element, ...animationOptions) => element.animate(...animationOptions);
+
+let updateBackground = (element, path, id) => {
+    element.style.backgroundImage = window.innerWidth < 800 ? `url('${path + id + "_small"}.jpg')` : `url('${path + id}.jpg')`;
+    addAnimation(element, [{
+        opacity: .1
+    }, {
+        opacity: 1
+    }], {
+        duration: 500
+    });
+}
+
+let setInnerData = (element, data) => element.innerHTML = data;
+
+let setData = (index) => {
+    let image = document.getElementById("tabImage"),
+        heading = document.querySelector(".tab__header"),
+        headerMenu = document.querySelector(".header"),
+        container = document.querySelector(".container"),
+        paragraph = document.querySelector(".tab__paragraph"),
+        period = document.querySelector(".statistics__text"),
+        position = document.querySelector("#position");
+
+    updateBackground(image, ASSETS_PATH, (CONTENT[index].heading).toLowerCase());
+    setInnerData(heading, CONTENT[index].heading);
+    setInnerData(position, CONTENT[index].position);
+    setInnerData(paragraph, CONTENT[index].paragraph);
+    addAnimation(container, [{
+        opacity: 0
+    }, {
+        opacity: 1
+    }], {
+        duration: (600)
+    });
+
+    setInnerData(period, CONTENT[index].period);
+    if (window.innerWidth > 800) {
+        updateBackground(headerMenu, ASSETS_PATH, `aside_${(CONTENT[index].heading).toLowerCase()}`)
+    };
+}
+
+window.addEventListener('resize', function(e) {
+    let image = document.querySelector(".header");
+    if (window.innerWidth <= 800) {
+        image.style.backgroundImage = 'none';
+    } else {
+        updateBackground(image, ASSETS_PATH, `aside_${(document.querySelector(".tab__header").textContent).toLowerCase()}`);
+    }
+});
+
+let renderContent = (data) => {
+    let menu = document.getElementById('menu'),
+        listItemClasses = ['menu__item', 'flex', 'center', 'full'];
+    fragment = new DocumentFragment();
+
+    data.forEach(element => {
+        let listItem = document.createElement('li');
```

Comment:
You can use const here because you don't reassign this value. You just change DOM node object properties.
More info [here](https://www.geeksforgeeks.org/javascript-const/) (example 6)

Reply:
Thank you for providing useful info:)